scripts/exports.sh

```
@@ -0,0 +1,43 @@
+# used to inject env variables into the docker-compose.yml file
+
+##################################
+### Client Environment Variables
+##################################
+export TOR_ADDITIONAL_DIR_AUTHORITY=""
+# TODO user should be able to set this
+export TOR_NICKNAME="el$(shuf -i 100000-999999 -n 1)"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)